test/jdk/java/awt/Desktop/ActiveWindowCloseTest.java

```diff
@@ -0,0 +1,124 @@
+
+/* Copyright (c) 2016, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
If this is a newly created test, then the copyright should contain only the present year.

Reply:
I have added the original date based on the above comment that’s we get during the PR review below
https://github.com/openjdk/jdk/pull/7512#issuecomment-1075171066